homeassistant/components/switcher_kis/__init__.py

```diff
@@ -72,12 +81,7 @@ async def stop_bridge(event: Event) -> None:
     return True
 
 
-async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_unload_entry(hass: HomeAssistant, entry: SwitcherConfigEntry) -> bool:
     """Unload a config entry."""
-    await async_stop_bridge(hass)
-
-    unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
-    if unload_ok:
-        hass.data[DOMAIN].pop(DATA_DEVICE)
-
-    return unload_ok
+    await entry.runtime_data.bridge.stop()
```

Comment:
This could be moved to `async_setup_entry` with `async_on_unload`.

Reply:
Great catch, this actually remove the need to store the bridge in `runtime_data`

Reply:
@thecode With that comment, I'm unsure if you want to make this change first? Or do you want to merge this PR first and make that change in a follow-up PR?

../Frenck

Reply:
Change first, I was just on the way to push the updates after testing :-)

Reply:
Awesome, marked the PR as a draft for now.

Reply:
lol that was quick 👍